desktop/components/main_layout/templates/react_index.jade

```diff
@@ -0,0 +1,7 @@
+extends index
+
+block head
+  != header
+
+block body
+  != body
```

Comment:
This whole file could potentially go away now, and we could just render `index` -- thoughts?